examples/article-access.amp.html

```diff
@@ -189,10 +189,11 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
           </div>
         </div>
 
-        <section amp-access="views > 0" amp-access-hide class="meter-section">
+        <section id="meter-notif" amp-access="views > 0" amp-access-hide class="meter-section">
           <template amp-access-template type="amp-mustache">
             You are viewing article {{views}} of {{maxViews}}!
```

Comment:
Nit for complete example: if views > maxViews it can say "You have read your {{maxViews}} free articles. Please login to read more"